### ISSUE TITLE
Facilitation of DCOP

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-data-register#484]
# Description
The Digital Code of Practice (DCOP) is a guide for everyone - public service employees and contractors - working with digital service delivery. By running a 1-hour workshop using the kit provided, it is hoped that everyone in the team/business area get the same practical guidance on modern digital services to citizens.

# Acceptance Criteria
Given: a team member attends the DCoP workshop
When: an invite is sent out
Then: they want to know what they are being called for, how the plan is adding value to the work they are already doing, ~~whether they are not learning something they already know their time is being used for something actionable/tangible that they can walk out the workshop with.~~

Given: a team leader
When: who is asking for members' time; sending out the meeting invite; investing time in to organizing, attending, and follow-up on the workshop themself; and taking the responsibility of trusting a member of the team to do that successfully.
Then: they know that that person can facilitate the workshop well and efficiently, can meet the leader's expectations, is given timely updates of the steps to be taken before, during, and after the workshop in writing, and is provided with tangible and mutually agreed upon outcomes.

# Development Checklist
**Step 0: Gather material**
- [x] Getting access to the facilitation kit from the DCOP team
- [x] Prepare a place/board to start taking shared notes
- [x] Scan through the material given with the kit
- [x] Understand what the intended outcome of this activity is
- [x] Meet @Dianadec and discuss a date to facilitate this - @kgorana - deadline: 2024-07-11

**Step 1: Invite**
- [x] Get to know the team leader's expectations
- [x] Finding out the optimal number of participants for the workshop
- [x] Decide on a date and time for arranging the workshop
- [x] Draft and send out a meeting invite to hold the time in participants' calendars
- [x] Share the project steps and workshop activities with Jess and Diana for their feedback

**Step 2: Demo**
- [x] Start preparing the workshop kit: 2024-07-23
- [x] Set up a time to demo the workshop with Diana and/or Jess - deadline: 2024-07-24
- [x] #527
- [x] Gather feedback from @JLWade and @Dianadec on changes to make for the final workshop
- [x] Incorporate the feedback in this ticket by updating the content here (that applies to the final workshop)

**Step 3: Distribution of agenda**
- [x] ASK HOW TO MAKE THE WORKSHOP ACCESSIBLE FOR THE ATTENDEES
- [x] Update the meeting invites with agenda and workshop materials for attendees' reference

**Step 4: Workshop**
- [x] Notes and timestamps in the [word file](https://bcgov.sharepoint.com/:w:/r/teams/08548-KaviJessAshley1on1s/Shared%20Documents/Kavi,%20Jess,%20Diana%201%20on%201s/Kavi%20-%20Rough%20Notes/Kavi%20-%20DCoP%20workshop%20presenter%20notes.docx?d=w0dd7b87fad984fd7b78bc76c74853c8a&csf=1&web=1&e=cbERvB)

**Step 5: Follow-up**
- [x] Submit DCoP post-facilitation survey

# Dependencies
**Blocked by**
- [x] Requesting the kit by emailing
- [x] Get in touch with @Dianadec to facilitate this workshop (and estimating a date)
- [x] Preparing the steps by @kgorana 

**Blocking**
- [x] Facilitation of the workshop
- [x] Post-event form to be sent to the DCOP team
- [x] People's awareness on the [DCOP's 10 principles](https://digital.gov.bc.ca/policies-standards/dcop/)

**Relevant documentation as reference**
- Miro board for behind-the-scenes notes: https://miro.com/app/board/uXjVK1CcmOM=/
- Miro board sent with the kit (will have to use this for the workshop): https://miro.com/app/board/uXjVK3RRwoY=/

# Definition of Ready
- [x] Acceptance criteria are included
- [x] Wireframes/Materials are included
- [x] Design / Solution is accepted by Product Owner and Scrum Master
- [x] Dependencies are identified (technical, business, regulatory/policy)
- [x] Story has been estimated (under 13 pts)

# Definition of Done
**In progress**
- [x] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
- [x] UI meets accessibility requirements
- [x] Work is traceable in GitHub

**PO Review**
- [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
- [ ] Reviewed and approved by Product Owner

# Notes
- https://digital.gov.bc.ca/policies-standards/dcop/
- https://compass.gww.gov.bc.ca/introducing-digital-code-of-practice/
- https://digital.gov.bc.ca/policies-standards/dcop/
  - [x] highlight the practices most relevant to our role and the work the team does
  - [x] list out the ways we are already working in alignment with these practices
  - [x] take these findings to the workshop and discuss it with the colleagues (audience)
  - [x] Attach the [one-pager](https://bcgov.sharepoint.com/:b:/r/teams/08548-KaviJessAshley1on1s/Shared%20Documents/Kavi,%20Jess,%20Diana%201%20on%201s/Kavi%20-%20WIP%20-%20DCOP%20Workshop%20Facilitation/One%20pager%20-%20facilitation%20kits%201-1.pdf?csf=1&web=1&e=udvcgT) with the invite

**Comment by kgorana** [comment]
using this file temporarily: https://app.mural.co/t/kavispesonalmuralworkspace2376/m/kavispesonalmuralworkspace2376/1712775878770/b81286142f69e2c9ae6fb3a76c473422be8e8357

**Comment by kgorana** [comment]
I need to work on this: https://miro.com/app/board/uXjVK1CcmOM=/?moveToWidget=3458764593973054155&cot=14

**Comment by kgorana** [comment]
post-facilitation survey is not submitted and all the tasks in this ticket are done - so moving to PO review :)

**Comment by LindsayMacfarlane** [comment]
@Dianadec - are there next steps associated with this ticket that need to chat about?

**Comment by LindsayMacfarlane** [comment]
Confirmed with Diana and Jess that this ticket is no longer needed. Closing.
